docs/snapshots.md

```diff
@@ -0,0 +1,54 @@
+# Index Snapshots
+
+This document describes index snapshots, what they are and how they can be created or updated.
+
+**Table of Contents**
+
+- [What Are Index Snapshots](#what-are-index-snapshots)
+- [Creating a Snapshot](#creating-a-snapshot)
+- [Updating a Snapshot](#updating-a-snapshot)
+    1. [New Index](#new-index)
+    2. [New Snapshot](#new-snapshot)
+
+## What Are Index Snapshots
+
+Index snapshots are images of the DPS index at a certain point in time.
```

Comment:
Not `at a certain point in time`. They are just images of the DPS index database, period. We can't create them while the indexer is running.